src/test/spec/v2_runner/test_file.rs

```diff
@@ -27,10 +34,12 @@ pub struct TestFile {
 
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "camelCase")]
+#[serde(deny_unknown_fields)]
```

Comment:
The v2 runner previously allowed unknown fields, which caused me some amount of confusion during testing when things I expected to fail didn't; I couldn't find a reason to be lax here so added the deny tag.

Reply:
Good call here, I usually include this tag on all of the test runner structs but must've missed this one. Tiny nit but can we move this into the previous `serde` tag (i.e. `#[serde(rename_all = "camelCase", deny_unknown_fields)]`)?

Reply:
Good catch, done.